zookeeper-server/src/test/java/org/apache/zookeeper/test/TestHammer.java

```diff
@@ -18,41 +18,43 @@
 
 package org.apache.zookeeper.test;
 
-
-import org.apache.zookeeper.CreateMode;
-import org.apache.zookeeper.ZooKeeper;
 import org.apache.zookeeper.AsyncCallback.VoidCallback;
+import org.apache.zookeeper.CreateMode;
 import org.apache.zookeeper.ZooDefs.Ids;
+import org.apache.zookeeper.ZooKeeper;
 import org.apache.zookeeper.common.Time;
 
 public class TestHammer implements VoidCallback {
 
-    static int REPS = 50000;
+    private static int reps = 50000;
```

Comment:
This is not just a style and formatting change as access level is changed. It seems OK, but I'd like to understand how check style works - does it do any reachability (possibly other types of static) analysis and make decision based on that?

Reply:
Yes it does some static analysis but I don't think it requests this change.

@tisunkun which is the rule that fires for this line?